AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/SubscriptionSync/ReconcileAndLocalSave/ModelReconciliationQueue.swift

```diff
@@ -12,6 +12,7 @@ import Combine
 enum ModelReconciliationQueueEvent {
     case started
     case paused
```

Comment:
Why don't we need the model name on these events, if we need it below?

Reply:
Valid point - they could be added, and probably will be at some point in time, but the only consumer of this class is currently `IncomingEventReconciliationQueueFactory`, and we aren't using the `started` and `paused` events.  These were initially added because I was planning to do unit testing around these class (which I have not gotten around to doing :))